CODEOWNERS

```diff
@@ -198,7 +197,8 @@ Kconfig*                                  @tejlmand
 /samples/cellular/sms/                    @trantanen @tokangas
 /samples/openthread/                      @rlubos @edmont @canisLupus1313 @maciejbaczmanski
 /samples/nrf_profiler/                    @pdunaj
-/samples/nrf_rpc/ps_client/               @nrfconnect/ncs-protocols-serialization
+/samples/nrf_rpc/protocols_serialization/ps_client/ @nrfconnect/ncs-protocols-serialization
```

Comment:
It seems a bit redundant to have a separate folder and prefix `ps_` (that means protocol serialization) for sample. It should be something one.

Reply:
Point taken I have changed it to `/samples/nrf_rpc/protocols_serialization`.

Reply:
was it updated already? I still see `protocols_serialization/ps_xxx` folders so not sure if I'm looking for the current version. Or if I understood the request here.

Reply:
I guess github keeps code to what initial comment was provided. However, it is marked as `Outdated`.

Reply:
@canisLupus1313, my comment was mostly to general folder structure. Probably, it was my mistake to leave this comment under this string. 
I meant, since you created folder `protocol_serialization` then prefix `ps_` is redundant. It is obviously so client and server of protocol serialization. I suggest:
`/samples/nrf_rpc/protocols_serialization/client`
`/samples/nrf_rpc/protocols_serialization/server`

Reply:
It was changed as suggested.